lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -698,7 +711,7 @@ def update_task_progress():
     report_store = ReportStore.from_config()
```

Comment:
Use this helper function instead: https://github.com/Stanford-Online/edx-platform/blob/master/lms/djangoapps/instructor_task/tasks_helper.py#L518